lib/Minz/View.php

```diff
@@ -202,36 +202,25 @@ public static function headStyle () {
 		$styles = '';
 
 		foreach(self::$styles as $style) {
-			$cond = $style['cond'];
-			if ($cond) {
-				$styles .= '<!--[if ' . $cond . ']>';
-			}
-
 			$styles .= '<link rel="stylesheet" ' .
 				($style['media'] === 'all' ? '' : 'media="' . $style['media'] . '" ') .
 				'href="' . $style['url'] . '" />';
 
-			if ($cond) {
-				$styles .= '<![endif]-->';
-			}
-
 			$styles .= "\n";
 		}
 
 		return $styles;
 	}
-	public static function prependStyle ($url, $media = 'all', $cond = false) {
+	public static function prependStyle($url, $media = 'all') {
```

Comment:
Something like this perhaps, @Alkarex ?
```suggestion
	/**
	 * Prepends a <link> element referencing stylesheet.
	 *
	 * @param string $url
	 * @param string $media
	 * @param bool $cond Conditional comment for IE, now deprecated and ignored
	 */
	public static function prependStyle($url, $media = 'all', $cond = false) {
```

Reply:
Would `@deprecated` work for a parameter?

Reply:
Not that I know of or can find at a glance. Unless you mean `@param deprecated $cond`; that'll probably work as a custom type.